web_search_autocomplete_prefetch/views/base_view.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record model="ir.ui.view" id="view_view_search_autocomplete">
+        <field name="model">ir.ui.view</field>
+        <field name="inherit_id" ref="base.view_view_search"/>
+        <field name="arch" type="xml">
+            <field name="name" position="attributes">
+                <attribute name="options">{'web_search_autocomplete_prefetch.disable': true}</attribute>
```

Comment:
Shouldn't this have a `t-translation="off"` ? A lot of the translations show up this value.